recipes/r-matchthem/meta.yaml

```
@@ -0,0 +1,90 @@
+{% set version = '1.2.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-matchthem
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/MatchThem_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/MatchThem/MatchThem_{{ version }}.tar.gz
+  sha256: 23420cf44cd573e274f9e59504fbc5e91af374f3138e055d7b0f433ba2546a8a
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - '*/R.dll'        # [win]
+    - '*/Rblas.dll'    # [win]
+    - '*/Rlapack.dll'  # [win]
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-matchit >=4.0.0
+    - r-weightit
+    - r-mice
+    - r-rlang
+    - r-survey
+  run:
+    - r-base
+    - r-matchit >=4.0.0
+    - r-weightit
+    - r-mice
+    - r-rlang
+    - r-survey
+
+test:
+  commands:
+    - $R -e "library('MatchThem')"           # [not win]
+    - "\"%R%\" -e \"library('MatchThem')\""  # [win]
+
+about:
+  home: https://github.com/FarhadPishgar/MatchThem
+  license: GPL-2.0-only
```

DESCRIPTION shows `GPL (>= 2)`, so

```suggestion
  license: GPL-2.0-or-later
```

---

Good catch!, updated.